allensdk/brain_observatory/ecephys/copy_utility/__main__.py

```diff
@@ -8,8 +8,13 @@
 import argschema
 
 from allensdk.config.manifest import Manifest
-from ._schemas import InputSchema, OutputSchema, available_hashers
-from allensdk.brain_observatory.argschema_utilities import write_or_print_outputs
+from ._schemas import (
+    SessionUploadInputSchema,
+    SessionUploadOutputSchema,
+    available_hashers
+)
+from allensdk.brain_observatory.argschema_utilities \
```

Comment:
we don't want to use anything from `argschema_utilities`
Normally, we would use:
https://github.com/AllenInstitute/argschema/blob/c7cf8fb53166fecd1d1b8f562e5b8c0c12f03a03/argschema/argschema_parser.py#L213